lms/migrations/versions/911d1f7da759_backfill_course_groupings.py

```diff
@@ -0,0 +1,73 @@
+"""
+Backfill grouping with courses based on rows from `course` and `group_info`.
+
+Revision ID: 911d1f7da759
+Revises: 2119e1c621de
+Create Date: 2022-03-25 11:35:19.760847
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "911d1f7da759"
+down_revision = "34f28d992b18"
+
+
+def upgrade():
+    conn = op.get_bind()
+
+    conn.execute(
+        """
+        INSERT INTO grouping
+            (
+                created,
+                application_instance_id,
+                authority_provided_id,
+                parent_id,
+                lms_id,
+                lms_name,
+                type,
+                settings,
+                extra
+            )
+        SELECT
+                -- Using a hard-coded value in the past to identify the rows added by the migration
+                '2022-03-25 00:00:00',
+                application_instances.id,
+                group_info.authority_provided_id,
+                -- No parent
+                NULL,
+                context_id,
+                context_title,
```

Comment:
```suggestion
                group_info.context_id,
                group_info.context_title,
```
? Is this right? With this many tables flying around it's nice to pin it down.